modules/utxo-lib/src/bitgo/legacysafe/index.ts

```diff
@@ -48,8 +48,9 @@ export function createLegacySafeOutputScript2of3(
   }
 
   pubkeys.forEach((key) => {
-    if (key.length !== 65) {
-      throw new Error(`Unexpected key length ${key.length}. Must use uncompressed keys.`);
+    // V1 Safe BTC wallets could contain both uncompressed and compressed pubkeys
+    if (key.length !== 65 && key.length !== 33) {
```

Comment:
```suggestion
    if (key.length !== keys[0].length) { throw new Error(`inconsistent key lengths`); }
    if (key.length !== 65 && key.length !== 33) {
```
    if (key.length !== 65 && key.length !== 33) {
```